Plugin/ConfigSectionPaymentPlugin.php

```diff
@@ -69,8 +69,12 @@ private function getKeys($configData)
     {
         $configFields = $configData['groups']['omise']['fields'];
 
-        $publicKeyIndex = ($configFields['sandbox_status']) ? 'test_public_key' : 'live_public_key';
-        $secretKeyIndex = ($configFields['sandbox_status']) ? 'test_secret_key' : 'live_secret_key';
+        $sandboxStatus = array_key_exists('value', $configFields['sandbox_status'])
```

Comment:
ah i see you will update it but 
i think for got about this comment 
```php
 // if sandbox status is updated the updated value will be under 'value' key else it won't have the value key
 ```

Reply:
@new4762 Thanks.